src/lib/components/End/index.js

```diff
@@ -11,22 +11,32 @@ const End = ({
   },
   handleChange,
 }) => (
-  <div>
-    <h2>End</h2>
-    <select
-      name="end.mode"
-      className="form-control"
-      value={mode}
-      onChange={event => handleChange(event)}
-    >
-      <option value="Never">Never</option>
-      <option value="After">After</option>
-      <option value="On date">On date</option>
-    </select>
+  <div className="container">
```

Comment:
You probably need only one `.container` on the top of the markup tree (there is already `.container` in AppContainer.js)

This additional container is In other places as well.

Reply:
`.container` provides some padding and I used it mostly in this purpose, for small spacing. Should I change it to '.p-1' for example?

Reply:
If you only care about padding, then using `p-1` is more semantically correct, let's go for it :)